installer/src/main/java/com/amazon/aws/partners/saasfactory/saasboost/SaaSBoostInstall.java

```diff
@@ -2228,17 +2228,22 @@ private String buildAndCopyWebApp() throws IOException {
         }
 
         //refresh the client
-        cfn = CloudFormationClient.create();
-
-        ListExportsResponse response = cfn.listExports(ListExportsRequest.builder()
-                .build());
+        cfn = CloudFormationClient.builder().region(AWS_REGION).build();
 
+        String nextToken = null;
         Map<String, String> exportsMap = new HashMap<>();
-        for (Export export : response.exports()) {
-            if (export.name().startsWith("saas-boost::" + envName)) {
-                exportsMap.put(export.name(), export.value());
+        do {
+            ListExportsResponse response = cfn.listExports(ListExportsRequest.builder()
+                    .nextToken(nextToken)
+                    .build());
+            nextToken = response.nextToken();
+            for (Export export : response.exports()) {
+                if (export.name().startsWith("saas-boost::" + envName)) {
+                    exportsMap.put(export.name(), export.value());
+                }
+                LOGGER.info("Export name: {}, Export value: {}", export.name(), export.value());
```

Comment:
Let's remove this to avoid extra information in log file that could reveal non saas-boost information.